packages/frontend-shared/src/gql-components/HeaderBarContent.spec.tsx

```diff
@@ -93,4 +93,141 @@ describe('<HeaderBarContent />', { viewportWidth: 1000, viewportHeight: 750 }, (
+        })
+
+        it('does not open when previously shown', () => {
+          mountWithSavedState({ promptsShown: { 'ci1': 1609459200000, 'orchestration1': 1609459200000 } })
+          it('does not open when projectId exists', () => {
+            mountWithSavedState()
+
+            cy.contains(defaultMessages.topNav.docsMenu.prompts.ci1.description).should('not.exist')
+          })
+
+          it('does not open when another prompt has been shown recently', () => {
+            mountWithSavedState({ promptsShown: { dashboard1: 1609891100000 } })
+
+            cy.contains(defaultMessages.topNav.docsMenu.prompts.ci1.description).should('not.exist')
+          })
+        })
+
+        it('does not open if "allowAutomaticPromptOpen" is not true', () => {
+          // we should be sure that, eg, in launchpad, this would not open up after a testing type is configured
+          cy.mountFragment(HeaderBar_HeaderBarContentFragmentDoc, {
+            onResult: (result) => {
+              if (!result.app.activeProject) {
+                return
+              }
+
+              result.app.activeProject.savedState = {
+                firstOpened: 1609459200000,
+                lastOpened: 1609459200000,
+                promptsShown: {},
+              }
+            },
+            render: (gqlVal) => <div class="resize overflow-auto border-current border-1 h-700px"><HeaderBarContent gql={gqlVal} show-browsers={true} /></div>,
+          })
+
+          cy.contains(defaultMessages.topNav.docsMenu.prompts.ci1.description).should('not.exist')
+        })
+      })
+    })
+
+    describe('the orchestration prompt', () => {
+      it('is not open by default', () => {
+        cy.get('.prompt-orchestration1').should('not.exist')
+      })
+
+      context('opens on click', () => {
+        beforeEach(() => {
+          cy.mountFragment(HeaderBar_HeaderBarContentFragmentDoc, {
+            render: (gqlVal) => <div class="resize overflow-auto border-current border-1 h-700px"><HeaderBarContent gql={gqlVal} show-browsers={true} /></div>,
+          })
+
+          cy.contains('Docs').click()
+          cy.contains('Run tests faster').click()
+        })
+
+        it('opens on menu item click', () => {
+          cy.contains(defaultMessages.topNav.docsMenu.prompts.orchestration1.title).should('be.visible')
+          cy.contains('Getting Started').should('not.exist')
+        })
+
+        it('is dismissible from X icon', () => {
+          cy.findAllByLabelText('Close').click()
+          cy.contains(defaultMessages.topNav.docsMenu.prompts.orchestration1.title).should('not.exist')
+        })
+
+        it('links to more information with expected utm params', () => {
+          cy.contains(
+            'a[href="https://on.cypress.io/smart-orchestration?utm_medium=CI+Prompt+1&utm_campaign=Learn+More"]',
+            defaultMessages.topNav.docsMenu.prompts.orchestration1.learnMore,
+          )
+          .should('be.visible')
```

Comment:
That is a lot of tests, love to see it!!

Reply:
Can't take too much credit, a lot of these originate in the current e2e tests for this functionality in desktop-gui :) but I do think it is worth making sure all existing tests for this are represented somewhere in the new codebase, if a feature is coming over 1:1. I did something similar with other parts of the nav where we already had tests.